checkin.js

```diff
@@ -11,34 +11,12 @@
 const AWS = require('aws-sdk');
 const async = require('async');
 const m_alServiceC = require('al-collector-js/al_servicec');
+const m_alAzCollectC = require('al-collector-js/azcollectc');
 const m_packageJson = require('./package.json');
 
 const AZCOLLECT_ENDPOINT = process.env.azollect_api;
 const COLLECTOR_TYPE = 'cwe';
 
-function checkCloudFormationStatus(event, callback) {
```

Comment:
Is this check moved into the `al-aws-collector-js`?

Reply:
Yep, its the only default heatlh check in al-aws-collector-js

https://github.com/alertlogic/al-aws-collector-js/blob/master/health_checks.js#L31-L51